meson.build

```diff
@@ -1,8 +1,12 @@
-project('dxvk', ['c', 'cpp'], default_options : ['cpp_std=c++1z'])
+project('dxvk', ['c', 'cpp'], default_options : ['cpp_std=c++17'])
```

Comment:
This should probably only be changed for MSVC as changing it for GCC could cause some older versions to not compile it even though they can.